backend/src/main/java/ddangkong/domain/balance/room/RoomStatus.java

```diff
@@ -0,0 +1,18 @@
+package ddangkong.domain.balance.room;
+
+public enum RoomStatus {
+    READY(false),
+    PROGRESS(true),
+    FINISH(false),
+    ;
+
+    private final boolean isGameProgress;
+
+    RoomStatus(boolean isGameProgress) {
+        this.isGameProgress = isGameProgress;
+    }
+
+    public boolean isGameProgress() {
+        return isGameProgress;
```

Comment:
```suggestion
        return this == PROGESS;
```

의견) boolean 변수가 의미있게 사용되지 않는 것 같아요 변수 제거 및 위와 같이 제안드립니다